SolWatchTests/PlanetTests.cs

```diff
@@ -30,5 +33,22 @@ public void AnomalyTest(string planetName, int year, int month, int day, float e
                 result,
                 Utilities.RadiansFromDegrees(angularAccuracyDegrees));
         }
+
+        [TestMethod]
+        [DataRow("Earth", 2023, 5, 20, 0.6934f)]
+        [DataRow("Earth", 2024, 8, 1, 1.9617f)]
+        [DataRow("Earth", 2025, 1, 6, 4.6798f)]
+        public void TrueLongitudeAtEpochTest(string planetName, int year, int month, int day, float expectedOutputRadians)
+        {
+            var testPlanet = testPlanets.Where(planet => planet.Name == planetName).FirstOrDefault();
```

Comment:
Let's update this how I updated my test method, using the .FirstOrDefault(Function predicate) overload to do the work of .Where(Function predicate).